extensions/wikia/RTE/RTE.class.php

```diff
@@ -61,6 +61,38 @@ public static function replacePlaceholder($var) {
 		$data = RTEData::get('placeholder', intval($var[1]));
 
 		if($data) {
+			wfProfileIn(__METHOD__);
+
+			// XW-2433 - hack starts
+			if ($data['type'] == 'double-brackets') {
+				global $wgTitle;
+
+				$wikitext = RTEData::get('wikitext', intval($data['wikitextIdx']));
+
+				// render template's wikitext
+				$parser = ParserPool::get();
+				$opts = ParserOptions::newFromContext( RequestContext::getMain() );
+
+				$parserOutput = $parser->parse( $wikitext, $wgTitle, $opts );
+
+				#var_dump(__METHOD__, $data, $wikitext, $parserOutput->getText());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)